MapboxSearch/ui/src/main/java/com/mapbox/search/ui/utils/extenstion/Context.kt

```diff
@@ -72,3 +83,51 @@ internal fun Context.resolveAttr(@AttrRes resId: Int): Int? {
         null
     }
 }
+
+/**
```

Comment:
Why do you need these changes? When you develop SDK it's better to keep everything non-public, until proven otherwise.

Reply:
Because it doesn't work otherwise:
```
> Task :sample:compileDebugKotlin FAILED
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/AddressAutofillUiActivity.kt: (29, 46): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/AddressAutofillUiActivity.kt: (175, 21): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/AddressAutofillUiActivity.kt: (183, 17): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/AddressAutofillUiActivity.kt: (194, 17): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/DiscoverActivity.kt: (36, 46): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/DiscoverActivity.kt: (113, 25): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/DiscoverActivity.kt: (132, 21): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/PlaceAutocompleteUiActivity.kt: (37, 46): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/PlaceAutocompleteUiActivity.kt: (209, 21): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/PlaceAutocompleteUiActivity.kt: (215, 17): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
e: .../MapboxSearch/sample/src/main/java/com/mapbox/search/sample/PlaceAutocompleteUiActivity.kt: (232, 17): Cannot access 'showToast': it is internal in 'com.mapbox.search.ui.utils.extenstion'
```

Reply:
If these functions are used only in sample module, it's better to move them there. I think these utilities have to be internal if they are really needed in ui module.

Api reference should be fixed anyway.

Reply:
Made them internal again